shared/get_data_format.m

```diff
@@ -36,8 +36,10 @@
     fprintf('Data is Micromed\n');
 elseif isBrainvision
     fprintf('Data is Brainvision\n');
+
```

Comment:
In the future, instead of having three boolean output variable which are mutually exclusive, better use a 'switch/case' scenario, i.e. output is a string, and code that use it use a: switch datatype; case Micromed.
Also check ft_datatype.m